server/command.go

```diff
@@ -95,10 +95,205 @@ func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
+	gitUser, _, err := githubClient.Users.Get(ctx, "")
+	if err != nil {
+		p.postCommandResponse(args, "Encountered an error getting your GitHub profile.")
+		return &model.CommandResponse{}, nil
+	}
+
+	text := fmt.Sprintf("You are connected to GitHub as:\n# [![image](%s =40x40)](%s) [%s](%s)", gitUser.GetAvatarURL(), gitUser.GetHTMLURL(), gitUser.GetLogin(), gitUser.GetHTMLURL())
+	p.postCommandResponse(args, text)
+	return &model.CommandResponse{}, nil
+}
+func handleHelp(p *Plugin, _ *plugin.Context, args *model.CommandArgs, _ []string, _ *github.Client, _ context.Context, _ *GitHubUserInfo) (*model.CommandResponse, *model.AppError) {
+	text := "###### Mattermost GitHub Plugin - Slash Command Help\n" + strings.Replace(COMMAND_HELP, "|", "`", -1)
+	p.postCommandResponse(args, text)
+	return &model.CommandResponse{}, nil
+}
+func handleEmpty(p *Plugin, _ *plugin.Context, args *model.CommandArgs, _ []string, _ *github.Client, _ context.Context, _ *GitHubUserInfo) (*model.CommandResponse, *model.AppError) {
+	text := "###### Mattermost GitHub Plugin - Slash Command Help\n" + strings.Replace(COMMAND_HELP, "|", "`", -1)
+	p.postCommandResponse(args, text)
+	return &model.CommandResponse{}, nil
+}
+func handleSettings(p *Plugin, _ *plugin.Context, args *model.CommandArgs, parameters []string, _ *github.Client, _ context.Context, userInfo *GitHubUserInfo) (*model.CommandResponse, *model.AppError) {
+	if len(parameters) < 2 {
+		p.postCommandResponse(args, "Please specify both a setting and value. Use `/github help` for more usage information.")
+		return &model.CommandResponse{}, nil
+	}
+
+	setting := parameters[0]
+	if setting != SETTING_NOTIFICATIONS && setting != SETTING_REMINDERS {
+		p.postCommandResponse(args, "Unknown setting.")
+		return &model.CommandResponse{}, nil
+	}
+
+	strValue := parameters[1]
+	value := false
+	if strValue == SETTING_ON {
+		value = true
+	} else if strValue != SETTING_OFF {
+		p.postCommandResponse(args, "Invalid value. Accepted values are: \"on\" or \"off\".")
+		return &model.CommandResponse{}, nil
+	}
+
+	if setting == SETTING_NOTIFICATIONS {
+		if value {
+			err := p.storeGitHubToUserIDMapping(userInfo.GitHubUsername, userInfo.UserID)
+			if err != nil {
+				mlog.Error(err.Error())
+			}
+		} else {
+			err := p.API.KVDelete(userInfo.GitHubUsername + GITHUB_USERNAME_KEY)
+			if err != nil {
+				mlog.Error(err.Error())
+			}
+		}
+
+		userInfo.Settings.Notifications = value
+	} else if setting == SETTING_REMINDERS {
+		userInfo.Settings.DailyReminder = value
+	}
+
+	err := p.storeGitHubUserInfo(userInfo)
+	if err != nil {
+		mlog.Error(err.Error())
+	}
+
+	p.postCommandResponse(args, "Settings updated.")
+	return &model.CommandResponse{}, nil
+}
+
+type CommandHandleFunc func(p *Plugin, c *plugin.Context, args *model.CommandArgs, parameters []string, githubClient *github.Client, ctx context.Context, userInfo *GitHubUserInfo) (*model.CommandResponse, *model.AppError)
```

Comment:
Every handler method returns `&model.CommandResponse{}, nil`. Can we unity the code?